roles/openshift_node/templates/node.service.j2

```diff
@@ -8,6 +8,7 @@ Wants={{ openshift.docker.service_name }}.service
 Documentation=https://github.com/openshift/origin
 Requires=dnsmasq.service
 After=dnsmasq.service
+{% if openshift.docker.use_crio %}Wants=cri-o.service{% endif %}
```

Comment:
@sjug true. The naming here is meant to not collide with the package service while still being obvious that it is indeed cri-o.